docs/excel/configure-your-add-in-to-use-the-browser-runtime.md

```diff
@@ -0,0 +1,114 @@
+---
+ms.date: 02/06/2020
+title: "Configure your Excel add-in to share the browser runtime (preview)"
+ms.prod: excel
+description: Configure your Excel add-in to share the browser runtime and run ribbon, task pane, and custom function code in the same runtime.
+localization_priority: Priority
+---
+
+# Configure your Excel add-in to share the browser runtime (preview)
+
+[!include[Running custom functions in browser runtime note](../includes/excel-shared-runtime-preview-note.md)]
+
+When running Excel on Windows or Mac, your add-in will run code for ribbon buttons, custom functions, and the task pane in separate JavaScript runtime environments. You can configure your Excel add-in to share code in a single browser runtime. This enables better coordination across your add-in for sharing code and data. It also enables you to run code when the document opens, or to run code while the task pane is closed. To configure your add-in to share the browser runtime, follow the instructions in this article.
+
+## Create the add-in project
+
+If you are starting a new project, follow these steps to use the Yeoman generator to create an Excel add-in project. Run the following command and then answer the prompts with the following answers:
+
+```command&nbsp;line
+yo office
+```
+
+- Choose a project type: **Excel Custom Functions Add-in project**
+- Choose a script type: **JavaScript**
+- What do you want to name your add-in? **My Office Add-in**
+
+![Screenshot of answering prompts from yo office to create the add-in project.](../images/yo-office-excel-project.png)
+
+After you complete the wizard, the generator creates the project and installs supporting Node components.
+
+## Configure the manifest
+
+Follow these steps for a new or existing project to configure it to use a shared context browser runtime.
+
+1. Start Visual Studio Code and open the **My Office Add-in** project.
+2. Open the **manifest.xml** file.
+3. Change the `<Requirements>` section to use **SharedRuntime** version **1.1** as shown in the following code.
+    
+    ```xml
+    <Requirements>
+    <Sets DefaultMinVersion="1.1">
+    <Set Name="SharedRuntime" MinVersion="1.1"/>
+    </Sets>
+    </Requirements>
+    ```
+    
+4. Find the `<VersionOverrides>` section, and add the following `<Runtimes>` section. The lifetime needs to be **long** so that the custom functions can still work even when the task pane is closed.
+    
+    ```xml
+    <VersionOverrides xmlns="http://schemas.microsoft.com/office/taskpaneappversionoverrides" xsi:type="VersionOverridesV1_0">
+      <Hosts>
+        <Host xsi:type="Workbook">
+        <Runtimes>
+          <Runtime resid="TaskPaneAndCustomFunction.Url" lifetime="long" />
```

Comment:
Consider putting "Contoso" or something like that in the resid value, like "Contoso.TaskPaneAndCustomFunction.Url" so that readers don't think that "TaskPaneAndCustomFunction.Url" is the required name. A resid can be any string they want, as long as they are consistent and use the same string everywhere.